releases/stellar/2025-05-ITS-v1.2.0.md

```diff
@@ -0,0 +1,88 @@
+# Stellar ITS v1.2.0
+
+|                | **Owner**                            |
+| -------------- | ------------------------------------ |
+| **Created By** | @ahram (<ahram@interoplabs.io>)   |
+| **Deployment** | @ahram (<ahram@interoplabs.io>)   |
+
+| **Network**          | **Deployment Status** | **Date**   |
+| -------------------- | --------------------- | ---------- |
+| **Devnet Amplifier** | TBD                   | 2025-05-12 |
+| **Stagenet**         | TBD                   | 2025-05-12 |
+| **Testnet**          | TBD                   | 2025-05-12 |
+| **Mainnet**          | TBD                   | TBD        |
+
+- [Crates](https://crates.io/crates/stellar-interchain-token-service/1.2.0)
+- [Releases](https://github.com/axelarnetwork/axelar-amplifier-stellar/releases/tag/stellar-interchain-token-service-v1.2.0)
+
+## Background
+
+- This is the v1.2.0 Stellar ITS release.
+
+Contract changes in the release:
+
+- emit data_hash instead of data in InterchainTransferSentEvent ([#342](https://github.com/axelarnetwork/axelar-amplifier-stellar/pull/342))
+- This is a minor bump since it breaks the event API for consumers.
```

Comment:
```suggestion
- ([#342](https://github.com/axelarnetwork/axelar-amplifier-stellar/pull/342)): Breaking change to the `InterchainTransferSentEvent` event in ITS. `data_hash` is emitted now instead of `data`.
```
describe the issue as breaking instead of making a separate note about it